ddtrace/llmobs/_integrations/__init__.py

```diff
@@ -13,4 +17,8 @@
     "GeminiIntegration",
     "LangChainIntegration",
     "OpenAIIntegration",
+    "VertexAIIntegration",
+    "extract_model_name",
```

Comment:
we can exclude these other methods from the `__all__`. The `__all__` only namespaces what gets imported when someone does a `from <module> import *` and doesn't affect `from llmobs._integrations import extract_model_name`.

```suggestion
```

also I don't think we need to re-import the helpers here either.